hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -860,23 +860,23 @@ public static void assertEnumCoverage(Class<? extends Enum<?>> enumClass) {
         }
     }
 
-    public static <E> void assertNotContains(Collection<E> collection, E expected) {
-        if (collection.contains(expected)) {
-            fail(format("Collection %s (%d) contained unexpected '%s'", collection, collection.size(), expected));
+    public static <E> void assertNotContains(Collection<E> actual, E notExpected) {
```

Comment:
There's one more `assertContains(Collection<E> collection, E expected)` method in the family, you could rename that one's arguments too.